src/main/java/org/breedinginsight/brapps/importer/services/processors/ExperimentProcessor.java

```diff
@@ -87,14 +84,7 @@
 public class ExperimentProcessor implements Processor {
 
     private static final String NAME = "Experiment";
-    private static final String EXISTING_ENV = "Cannot create new observation unit %s for existing environment %s.<br/><br/>" +
-            "If you’re trying to add these units to the experiment, please create a new environment" +
-            " with all appropriate experiment units (NOTE: this will generate new Observation Unit Ids " +
-            "for each experiment unit).";
-    private static final String MISSING_OBS_UNIT_ID_ERROR = "Experiment Units are missing Observation Unit Id.<br/><br/>" +
-            "If you’re trying to add these units to the experiment, please create a new environment" +
-            " with all appropriate experiment units (NOTE: this will generate new Observation Unit Ids " +
-            "for each experiment unit).";
+    private static final String MISSING_OBS_UNIT_ID_ERROR = "Experimental entities are missing ObsUnitIDs. You must append file with the appropriate ObsUnitIDs or specify a new environment for these entities to proceed";
```

Comment:
```suggestion
    private static final String MISSING_OBS_UNIT_ID_ERROR = "Experimental entities are missing ObsUnitIDs";
```

This will be interpolated into `"Error detected in file, ${fileName}. ${backendErrorMessage}. Import cannot proceed."` by bi-web, to produce the desired message, `"Error detected in file, XXX.xls.  Experimental entities are missing ObsUnitIDs. Import cannot proceed."`. See row 15 in the [source of truth](https://breedinginsight.atlassian.net/l/cp/wHDdzYFb).